src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs

```diff
@@ -154,7 +155,7 @@ public void Stop()
 
             if (await _clientSettingsRepository.IsOffchainClient(queueMessage.ToClientid))
             {
-                if (!await _clientAccountsRepository.IsTrusted(queueMessage.ToClientid))
+                if (!(await _clientAccountService.IsTrustedAsync(queueMessage.ToClientid)).Value)
```

Comment:
Is IsTrustedAsync always returns not null? Why it nullable then.

Reply:
IsTrusted controller always returns bool (not null) if request result is 200 OK.  NotFound and BadRequest are valid results, and autorest makes API response nullable to handle all situations. 
I'll remove value check at caller side.